cmd/kubeless/deploy.go

```diff
@@ -109,6 +109,14 @@ var deployCmd = &cobra.Command{
 			funcDeps = string(bytes)
 		}
 
+		if runtime == "" && runtimeImage == "" {
+			logrus.Fatal("Either `--runtime` or `--runtime-image` flag must be specified.")
+		}
+
+		if runtime != "" && handler == "" {
```

Comment:
I would say `if runtimeImage == "" && handler == ""`  because it can be possible to specify a runtime with a runtime image and skip the handler

Reply:
Dont you think its already handled? Either `--runtime` or `--runtime-image` must specified. If `--runtime` is specified then handler must be specified.

Reply:
ah, yes, it is the same, sorry